app/javascript/modules/listingDetailsLottery/ListingDetailsLotteryPreferencesEducator.tsx

```diff
@@ -0,0 +1,195 @@
+import React from "react"
+import { Heading, t } from "@bloom-housing/ui-components"
+import type { RailsLotteryResult } from "../../api/types/rails/listings/RailsLotteryResult"
+import { defaultIfNotTranslated, renderMarkup } from "../../util/languageUtil"
+import { preferenceNameHasVeteran } from "../../util/listingUtil"
+
+export interface ListingDetailsLotteryPreferencesProps {
+  lotteryBucketsDetails: RailsLotteryResult
+  isEducatorOne: boolean
+}
+interface PreferencesProps {
+  unitsAvailable: number
+  totalSubmittedApps: number
+  preferenceName: string
+  numVeteranApps: number
+  lastPref: boolean
+}
+
+const LotteryPreferences = ({
+  unitsAvailable,
+  totalSubmittedApps,
+  preferenceName,
+  numVeteranApps,
+  lastPref,
+}: PreferencesProps) => (
+  <div>
+    <div className="px-8">
+      <Heading className="font-sans font-semibold text-xs tracking-wide uppercase" priority={3}>
+        {defaultIfNotTranslated(
+          `listings.lotteryPreference.${preferenceName}.title`,
+          preferenceName
+        )}
+      </Heading>
+      <p className="text-sm">{t("lottery.upToXUnitsAvailable", { units: unitsAvailable ?? 0 })}</p>
+      {numVeteranApps ? (
+        <>
+          <p className="text-gray-700 text-sm">
+            {t("lottery.numberApplicantsQualifiedForPreference.veteran", {
+              number: numVeteranApps,
+              plural: numVeteranApps > 1 ? "s" : "",
+            })}
+          </p>
+          <p className="text-gray-700 text-sm">
+            {t("lottery.numberApplicantsQualifiedForPreference.nonVeteran", {
+              number: totalSubmittedApps,
+              plural: numVeteranApps > 1 ? "s" : "",
+            })}
+          </p>
+        </>
+      ) : (
+        <p className="text-gray-700 text-sm">
+          {t("lottery.numberApplicantsQualifiedForPreference", {
+            number: totalSubmittedApps,
+            plural: numVeteranApps > 1 ? "s" : "",
+          })}
+        </p>
+      )}
+    </div>
+    <hr className={lastPref ? "mt-4" : "my-4"} />
```

Comment:
thought: there's probably some way to remove the need for the `lastPref` prop by rethinking the styling implementation, but it's not worth the effort imo.

Reply:
agreed! Was thinking of this as a quick implementation just for SCV, anticipating a bigger lottery modal redesign down the line